File: Feitos em C Sharp/3037.cs
using System;
namespace ConsoleApp12 {
    class Program {
        static void Main(string[] args) {
			int times = int.Parse(Console.ReadLine());
			for (int i = 0; i < times; i++) {
				int quantJoao = 0, quantMaria = 0;
				for (int j = 0; j < 3; j++) {
					string[] vet = Console.ReadLine().Split();
					int x = int.Parse(vet[0]);
					int d = int.Parse(vet[1]);
					quantJoao += (x * d);
				}
				for (int j = 0; j < 3; j++) {
					string[] vet = Console.ReadLine().Split();
					int x = int.Parse(vet[0]);
					int d = int.Parse(vet[1]);
					quantMaria += (x * d);
				}
				if (quantJoao > quantMaria) {
					Console.WriteLine("JOAO");
				}
				else {
					Console.WriteLine("MARIA");
				}
			}
		}
    }
}

File: Feitos em Python 3/3037.py
times=int(input())
i=0
while i<times:
    quantJoao=0
    quantMaria=0
    j=0
    while j<3:
        vet=input().split()
        x=int(vet[0])
        d=int(vet[1])
        quantJoao+=(x*d)
        j+=1
    j=0
    while j<3:
        vet=input().split()
        x=int(vet[0])
        d=int(vet[1])
        quantMaria+=(x*d)
        j+=1
    if quantJoao>quantMaria:
        print("JOAO")
    else:
        print("MARIA")
    i+=1